funcx_sdk/funcx/sdk/client.py

```diff
@@ -381,9 +376,8 @@ def map_run(self, *args, endpoint_id=None, function_id=None, asynchronous=False,
         if r.http_status != 200:
             raise Exception(r)
 
-        if 'task_uuids' not in r:
-            raise MalformedResponse(r)
-
+        if r.get("status", "Failure") == "Failure":
```

Comment:
Has the response changed ?

Reply:
No, but it seemed like it made more sense to check for a "Failure" status, rather than have a lack of `task_uuids` imply failure. I'd be happy to switch back if you like the other way better.

Reply:
This is alright.